\documentclass{article}
\textheight24cm\textwidth16cm\topmargin-1cm
\oddsidemargin0cm\evensidemargin0cm

\usepackage{amsmath,amssymb}
% \usepackage[dvipdfmx]{graphicx}
\usepackage{graphicx}
\usepackage{subfigure}

\usepackage{url}
\usepackage{bm}
\usepackage{setspace} 
\usepackage{subfigure}

% \usepackage{algorithmic}
% \usepackage{algorithm}

\newtheorem{thm}{Theorem}
\newtheorem{dfn}[thm]{Definition}
\newtheorem{lem}[thm]{Lemma}
\newtheorem{prop}[thm]{Proposition}
\newtheorem{cor}[thm]{Corollary}
\newcommand{\proof}{\noindent Proof.\ \ }
% \renewcommand{\algorithmiccomment}[1]{// #1}


%\pagestyle{empty}


\begin{document}

% \baselineskip 6.5mm

\vspace*{1.0cm}
{\LARGE \bf Supplementary material}
\vspace*{0.25cm}


\section*{Experiments on synthetic data}

We begin by illustrating the ability of our method to extract ``true'' mutation signatures on simulated data. In these simulations a mutation pattern
is defined by a substitution and the $\pm 2$ flanking bases.
We generated a set of mutations changing the number of cancer genomes ($I=10, 25, 50, 100$),  and the number of mutations for each cancer genome ($J=10, 25, 50, 100, 250, 500, 1000$).
The number of mutation signatures were set to $K=5$ including background mutation ratio.
The mutation feature parameters and membership parameters were generated by Dirichlet distribution,
\begin{equation}
\bm{f}_{k,l} \sim \text{Dir} (\alpha \bm{1} ),\ k = 1, \cdots, K,\  l = 1, \cdots, L.
\end{equation}
 \begin{equation}
 \bm{q}_{i, k} \sim \text{Dir} (\gamma \bm{1} ),\ i = 1, \cdots, I,
 \end{equation}
where $\alpha$ and $\gamma$ control the amount of dispersion 
for the mutation signature parameters and membership parameters, respectively.
For example, when $\gamma$ is small, most samples will have most of their
mutations coming from a single signature (but not the same signature for each sample). When $\gamma$ is large most samples will have mutations coming from all signatures in roughly equal proportions.

As Supplementary Figure 1(a) shows, we estimate the mutation signatures very accurately overall (see Supplementary Figure 1(b) for an example).
As expected, accuracy improves as we increase the number of cancer genomes or mutations.
Also, as the mutation feature dispersion decreases ($\alpha$ increases), accuracy decreases.
This is expected: as $\alpha$ increases the signatures will become more ``fuzzy", with
probability mass spread over a large number of mutation patterns, and so harder to infer
(In other words, larger $\alpha$ corresponds to a less informative motif).
In contrast, accuracy was relatively insensitive to 
individual membership dispersion ($\gamma$), and estimates remain accurate even when most individuals have mutations in many different signatures.

In most cases, the log-likelihood stopped increasing at $K=5$ mutation signatures, 
whereas the standard error of the estimated parameters started increasing past $K=5$ (see Supplementary Figure 1(c)). We conclude that examining
the trade-off between likelihood and standard-errors may be a helpful guide to selecting an appropriate number of mutation signatures.




\section*{Experiment on UCUT data with the various number of mutation signatures}

We analyzed the data using our new model, with increasing number of mutation signatures $K=2,3,\dots$. 
Results are shown in Supplementary Figure 2.
As expected, the likelihood increased with $K$. Bootstrap-errors started to increase at $K = 5$ (Supplementary Figure 3).

With $K = 2$ (Supplementary Figure 2 (a)) we observed a mutation signature that appears to correspond to AA  (T $>$ A substitutions at CpTpG sites with strong transcription strand specificity).
Increasing to  $K=3$ introduced an additional mutation signature corresponding to the APOBEC enzyme (C $>$ [AGT] at TpCpN sites) (Figure \ref{UTUC:APOBEC_ind5_sig}, \ref{UTUC:AA_ind5_sig}, Supplementary Figure 2 (b)).
Increasing to $K = 4$ introduced an additional signature (T $>$ A at NpTpN sites with strong strand specificity) 
that is somewhat similar to the AA signature 
(Supplementary Figure 2 (c)). 
The membership in the AA signature and this ``AA-like" signature were correlated ($R = 0.77$; Supplementary Figure 4). 
This additional signature may be just making up for the residual of the AA signature 
which the original AA signature could not explain due to a slight deviance of the probabilistic model.

The strong correlation among estimated membership parameters started to be shown at $K = 4$.
Considering all these factors together, $K = 3$ seems to be a reasonable choice in terms of the interpretability,
and we adopted $K = 3$.


\section*{Derivation of EM algorithm}


The complete log-likelihood including missing data $\{ \bm{z}_i \}$ for the proposed model is
\begin{equation*}
\sum_{i = 1}^I \sum_{j=1}^J \sum_{k=1}^K I(z_{i,j} = k) \Bigl( \sum_{l = 1}^L \log f_{k, l, x_{i,j,l}} + \log q_{i,k} \Bigr).
\end{equation*}
Here, we introduce the variable for conditional probability for $z_{i,j}$ given the parameters and the mutation features $\bm{x}_{i,j}$,
\begin{equation*}
\theta_{i,k, \bm{m}} = \Pr \bigl( z_{i,j} = k \big| \bm{x}_{i, j} = \bm{m}, \{ \bm{f}_{k, l} \}, \{  \bm{q}_i \} \bigr)
\end{equation*}
Note that this conditional probability just depends on the value of mutation feature $\bm{m} = (m_1, \cdots, m_L)$, not on the index $j$.
Then, the expected complete log-likelihood augmented by Lagrange multipliers is calculated as
\begin{equation*}
\sum_{i=1}^I \sum_{\bm{m}} g_{i, \bm{m} } \sum_{k=1}^K \theta_{i, k, \bm{m}}   \Bigl( \sum_{l = 1}^L \log f_{k, l, m_l} + \log q_{i,k} \Bigr)
+ \sum_{k=1}^K \sum_{l=1}^L \tau_{k,l} \bigl( 1 - \sum_{p=1}^{M_l} f_{k,l,p} \bigr)
+ \sum_{i=1}^I \rho_i \bigl( 1 - \sum_{k=1}^K q_{i,k} \bigr).
\end{equation*}
Differentiating it leads to following stationary equations: 
\begin{align*}
\sum_{i=1}^I \sum_{\bm{m}: m_l = p} g_{i, \bm{m}} \theta_{i,k,\bm{m}} - \tau_{k,l} f_{k,l,p} & = 0,\ (p=1, \cdots, M_l, k=1, \cdots, K,\ l=1, \cdots, L)., \\
\sum_{\bm{m}} g_{i,\bm{m}} \theta_{i,k,\bm{m}} - \rho_i q_{i, k} & = 0,\ (k = 1, \cdots, K, i = 1, \cdots, I).
\end{align*}
Then, by eliminating Lagrange multipliers, updating rules can be obtained.



\section*{Relationship with nonnegative matrix factorization}

First, for ease of explanation, let assume that the full representation'' representation ($L = 1$) is used.
Suppose that each $\bm{m}$ has unique appropriate index from 1 to  $| \bm{M} | = \prod_{l=1}^L M_l$ (the number of possible mutation patterns),
so that $\bm{m}$ can be indices of matrices.

Let $G = \{ g_{i, \bm{m}} \}$ denote the $I \times | \bm{M} | $ matrix, 
where $g_{i, \bm{m}}$ is the number of mutations whose mutation patters are equal to $\bm{m}$ in the $i$-th cancer genome.
Nonnegative matrix factorization aims to find low rank decomposition, 
$G \sim \tilde{Q} F$,
where $\tilde{Q} = \{ \tilde{q}_{i,k} \}$ and $F = \{ f_{k, \bm{m}} \}$ are nonnegative matrix, 
and row vectors of $F$ are often restricted to be sum to one. 
We used the notation $\tilde{Q}$ instead of $Q$
to represent that the row vectors of $\tilde{Q}$ are not normalized to sum to one in general.

For solving NMF, 
the previous study (Lee et al. 2000) used the following updating rule:
\begin{equation*}
f_{k,m} \gets f_{k,m} \frac{ ( \tilde{Q}^T G)_{k,m} }{ ( \tilde{Q}^T \tilde{Q} F  )_{k,m} },\  \
\tilde{q}_{i,k} \gets \tilde{q}_{i,k}  \frac{ (G F^T)_{i,k} }{ ( \tilde{Q} F F^T )_{i,k} },
\end{equation*}
that reduces the {\it Euclidean distance} $|| G - \tilde{Q} F ||$. 
Therefore, the optimization problem for the existing approach is
\begin{equation}
\begin{aligned}
& \text{minimize}
& & ||G - \tilde{Q} F || \\
& \text{subject to}
& & \sum_{ \bm{m} } f_{k, \bm{m}} =  1, \; k = 1, \cdots, K \\ &
& & f_{k, \bm{m}} \geq 0, \; k = 1, \cdots, K,\ \bm{m} \in M \\ & 
& & \tilde{q}_{i, k} \geq 0, \; i = 1, \cdots, I,\ k = 1, \cdots, K.
\end{aligned}
\end{equation}

On the other hand, there is another type of updating rule:
\begin{equation*}
f_{k,m} \gets f_{k,m} \frac{ \sum_i \tilde{q}_{i,k} g_{i,m} / ( \tilde{Q} F)_{i,m} }{ \sum_i \tilde{q}_{i,k} },
\end{equation*}
\begin{equation*}
\tilde{q}_{i,k} \gets \tilde{q}_{i,k}  \frac{ \sum_m f_{k,m} g_{i,m} / ( \tilde{Q} F)_{i,m} }{ \sum_m f_{k,m} }.
\end{equation*}
that reduces the Kullback-Liebler Divergence:
\begin{equation*}
KL(G ||  \tilde{Q} F) = \sum_{i,m} \Bigl( g_{i,m} \log  \frac{ g_{i,m} }{ (\tilde{Q} F)_{i,m} } - g_{i,m} + ( \tilde{Q} f )_{i,m} \Bigr).
\end{equation*}

In general cases including the independent representation, 
there is restrictions $f_{k, \bm{m}} = \prod_l f_{k, l, m_l}$ by smaller set of parameters.
Let us consider the following optimization problem with the Kullback-Liebler Divergence and the restrictions on $F$:
\begin{equation}
\begin{aligned}
& \text{minimize}
& & KL(G ||  \tilde{Q} F) \\
& \text{subject to}
& & f_{k, \bm{m}} = \prod_l f_{k, l, m_l}, \; k = 1, \cdots, K,\ \bm{m} \in M \\ & 
& & f_{k, l, p} \geq 0, \; k = 1, \cdots, K,\ \bm{m} \in M \\ & 
& & \tilde{q}_{i, k} \geq 0, \; i = 1, \cdots, I,\ k = 1, \cdots, K.
\end{aligned}
\label{KL}
\end{equation}

In fact, this is equivalent to the proposed method, whose optimization problem can be written as:
\begin{equation}
\begin{aligned}
& \text{maximize}
& & L(Q, F | G) \bigl(= \sum_{i,m}  g_{i,m} \log (Q F)_{i,m} \bigr)  \\
& \text{subject to}
& & f_{k, \bm{m}} = \prod_l f_{k, l, m_l}, \; k = 1, \cdots, K,\ \bm{m} \in M \\ & 
& & f_{k, l, p} \geq 0, \; k = 1, \cdots, K,\ \bm{m} \in M \\ & 
& & \sum_k  q_{i, k} =  1, \; i = 1, \cdots, I \\ &
& & q_{i, k} \geq 0, \; i = 1, \cdots, I,\ k = 1, \cdots, K.
\end{aligned}
\label{likelihood}
\end{equation}

\begin{prop}
When $(Q, F) = (Q^*, F^*)$ is an optimal solution of the optimization problem (\ref{likelihood}), 
then $(\tilde{Q}, F) = (R^* Q^*, F^*)$ is an optimal solution of the optimization problem (\ref{KL}).
On the other hand, when $(\tilde{Q}, F) = (\tilde{Q}^*, F^*)$ is an optimal solution of the optimization problem (\ref{KL}),
then $(Q, F) = (R^{*-1} \tilde{Q}^*, F^*)$ is an optimal solution of the optimization problem (\ref{likelihood}),
where $R^* = \text{diag} (r_1^*, \cdots, r_I^*), r_i^* = \sum_{\bm{m}} g_{i,\bm{m}}, i = 1, \cdots, I$.
\end{prop}
\proof
This is because
\begin{equation*}
KL(G ||  \tilde{Q} F) = -  \sum_i \Bigl( ( \sum_{m} g_{i, m} ) \log \tilde{r}_i - \tilde{r}_i \Bigr)  - L(Q, F | G) + \text{(constant value)},
\end{equation*}
where $Q$ is row-normalized matrix for $\tilde{Q}$, $\tilde{r}_i = \sum_k q_{i, k}$ for each $i$, 
and $( \sum_{m} g_{i, m} ) \log \tilde{r}_i - \tilde{r}_i$ takes its maximum at $\tilde{r}_i = r_i^*$.
\hfill $\square$

% \bibliographystyle{natbib}
%\bibliographystyle{achemnat}
%\bibliographystyle{plainnat}
%\bibliographystyle{abbrv}
%\bibliographystyle{bioinformatics}
\bibliographystyle{plain}
\bibliography{document}


\clearpage



\begin{table}[!ht]
\caption{
{\bf Example of representation for mutation patterns (substitution patterns and one 5' and 3' bases)}
In the independent representation, the elements of vector show substitution patterns, 5' adjacent bases and 3' adjacent bases, respectively.
For substitution pattens, 1 to 6 values are assigned to C$>$A, C$>$G, C$>$T, T$>$A, T$>$C and T$>$G in this order.
For 5' and 3' adjacent bases, 1 to 4 values are assigned to A, C, G and T.
Note that the original base is fixed to C or T to remove the redundancy of complement sequences.
}
\begin{center}
\begin{tabular}{|c|c|c|} \hline
mutation pattern & full model & independent model \\ \hline
$L$ & 1  & 3  \\
$\bm{M}$ & (96)  & (6, 4, 4) \\  \hline
ApCpA $\to$ ApCpA & (1)  & (1, 1, 1)  \\ 
ApCpC $\to$ ApApC & (2)  & (1, 1, 2)  \\ 
ApCpG $\to$ ApApG & (3)  & (1, 1, 3)  \\
ApCpT $\to$ ApApT & (4)  & (1, 1, 4)  \\
CpCpA $\to$ CpApA & (5)  & (1, 2, 1) \\
$\cdots$ & $\cdots$ & $\cdots$ \\
ApCpA $\to$ ApGpA & (17)  & (2, 1, 1) \\
$\cdots$ & $\cdots$ & $\cdots$ \\
TpTpT $\to$ TpGpT & (96)  & (6, 4, 4) \\ \hline
\end{tabular}
\end{center}
\label{tab_rep}
\end{table}



\clearpage


\begin{table}
\caption{The number of mutation signatures selected for each cancer type.
}
\begin{center}
\begin{tabular}{|c|c|} \hline
cancer type & \# mutation signature \\ \hline
AML & 2 \\ \hline
ALL & 3 \\ \hline
Bladder & 3 \\ \hline
Breast & 4 \\ \hline
Cervix & 3 \\ \hline
CLL & 3 \\ \hline
Colorectum & 5 \\ \hline
Esophageal & 4 \\ \hline
Glioblastoma & 3 \\ \hline
Glioma-Low-Grade & 3 \\ \hline
Head-and-Neck & 5 \\ \hline
Kidney-Chromophobe & 3 \\ \hline
Kidney-Clear-Cell & 3 \\ \hline
Kidney-Papillary & 3 \\ \hline
Liver & 3 \\ \hline
Lung-Adeno & 4 \\ \hline
Lung-Small-Cell & 4 \\ \hline
Lung-Squamous & 3 \\ \hline
Lymphoma-B-Cell & 4 \\ \hline
Medulloblastoma & 3 \\ \hline
Melanoma & 4 \\ \hline
Myeloma & 3 \\ \hline
Neuroblastoma & 3 \\ \hline
Ovary & 3 \\ \hline
Pancreas & 4 \\ \hline
Pilocytic-Astrocytoma & 4 \\ \hline
Prostate & 2 \\ \hline
Stomach & 5 \\ \hline
Thyroid & 4 \\ \hline
Uterus & 4 \\ \hline
\end{tabular}
\end{center}
\label{tab_rep}
\end{table}

\clearpage


\begin{figure}

\subfigure[The accuracy of the proposed approach for the simulated data when changing the number of samples, mutations, 
and the amounts of dispersion parameters ($\alpha$ and $\gamma$) for the mutation features and signature distribution parameters.]{%
\includegraphics[width=16cm,height=6cm]{simulation_result.eps}
\label{simulation_facet}
}

\subfigure[An example of the relationship between true and estimated mutation signatures in the simulation.
In this example, the numbers of cancer genomes and mutations for each cancer genome are 25 and 100, respectively,
and the parameters $\alpha$ and $\gamma$ are both set to 1. 
From this figure, we can see that fairly accurate estimation is possible even with moderate numbers of cancer genomes and mutations]{%
\includegraphics[width=15cm,height=5.5cm]{simulation_signature_example.eps}
\label{example_signature.pd}
}

\subfigure[The log-likelihood, bootstrap-errors and maximum correlation values among estimated membership parameters 
for several numbers of signatures $K$ in the trial of the above figure.]{%
\includegraphics[width=12cm,height=4cm]{simulation_stat.eps}
\label{example_summary.pd}
}

\caption{The summary of simulation study.}

\end{figure}


\clearpage


\begin{figure}
\centering

\subfigure[$K = 2$]{%
  \includegraphics[height=6cm,width=15cm,clip]{UTUC_signature_K2.eps}
  \label{UTUC:sigK2_sig}}
  
  
\subfigure[$K = 3$]{%
  \includegraphics[height=6cm,width=15cm,clip]{UTUC_signature_K3.eps}
  \label{UTUC:sigK3_sig}}
  

\subfigure[$K = 4$]{%
  \includegraphics[height=6cm,width=15cm,clip]{UTUC_signature_K4.eps}
  \label{UTUC:sigK4_sig}}

  
\caption{The result of estimated mutation signatures and membership parameters for UTUC data 
when changing the number of mutation signatures $K$.}
\label{UTUC_multiK}
\end{figure}

\clearpage

\begin{figure}
\centering
\includegraphics[width=15cm,height=5cm]{UTUC_stat.eps}
\caption{The log-likelihood, bootstrap-errors and maximum correlation values among estimated membership parameters for several numbers of signatures $K$ in UTUC data.}
\label{UTUC_stat}
\end{figure}

\clearpage

\begin{figure}
\centering
\includegraphics[width=10cm,height=10cm]{UTUC_AA_AAlike_cor.eps}
\caption{The relationships between the two membership parameters, AA (the first signature in the Supplementary Figure 3) and AA\_like (the second signature in the Supplementary Figure 3) signatures.}
\label{UTUT_AA_AAlike}
\end{figure}


\clearpage

\begin{figure}
\centering
\includegraphics[width=4cm,height=8cm]{UTUC_APOBEC_twoFivePrime.eps}
\caption{The frequencies of bases at two 5' to the mutated site for the APOBEC mutation signatures obtained in UTUC data using the independent and full models.}
\label{UTUC_APOBEC_twoFivePrime}
\end{figure}


\clearpage

\begin{figure*}

\subfigure[APOBEC signatures obtained in each cancer type]{%
  \includegraphics[width=15cm,height=7.5cm]{signatureList_APOBEC.eps}
  \label{APOBEC_sig_list}}
  
\subfigure[Smoking signature  in each cancer type]{%
  \includegraphics[width=15cm,height=2.5cm]{signatureList_LUNG.eps}
  \label{LUNG_sig_list}}
  
\subfigure[POLE1 signatures in each cancer type]{%
  \includegraphics[width=7.5cm,height=2.5cm]{signatureList_POLE1.eps}
  \label{POLE1_sig_list}}
\quad
\subfigure[POLE2 signature in each cancer type]{%
  \includegraphics[width=7.5cm,height=2.5cm]{signatureList_POLE2.eps}
  \label{POLE2_sig_list}}
  
\subfigure[UV signature in each cancer type]{%
  \includegraphics[width=7.5cm,height=2.5cm]{signatureList_UV.eps}
  \label{UV_sig_list}}
\quad
\subfigure[Unknown signature obtained in lung small cell carcinomas and stomach cancers]{%
  \includegraphics[width=7.5cm,height=2.5cm]{signatureList_LMST.eps}
  \label{LMST_sig_list}}
    
% \subfigure[Signature intensities at two 5' to the mutated site]{%
% \includegraphics[width=8cm,height=5cm]{APOBEC_two5prime.eps}
 % \label{APOBEC_twofive}}

\caption{(a) APOBEC signatures obtained in 11 cancer types, 
(b, c, d, e) Several signatures having prominent characteristics at 5'  or 3' to the mutated sites,
(f) the frequencies of bases at two 5' to the mutated site. 
}
\label{nature2013_apobec}
\end{figure*}

\clearpage


\begin{figure}
\centering

\subfigure[The low grade glioma specific signature in the previous study (the ``Signature 14'' in Alexandrov et al. (2013)]{%
  \includegraphics[height=3cm,width=12cm,clip]{example96_Glioma-Low-Grade.eps}
  \label{LGG_nature2013_sig}}
  
\subfigure[Estimated membership parameter by the proposed method]{%
  \includegraphics[height=4cm,width=12cm,clip]{Glioma-Low-Grade_membership.eps}
  \label{LGG_membership}}
\quad
\subfigure[Estimated membership parameter by the proposed method in log scale]{%
  \includegraphics[height=4cm,width=12cm,clip]{Glioma-Low-Grade_membership_log.eps}
  \label{LGG_membership_log}}

  \caption{(a) The barplot is divided by 6 substitution patterns. In each division, 
16 bars show joint probabilities of 16 combinations of the immediate 5' and 3' bases.
(b, c) We have selected top 100 cancer samples according to the number of mutation.
The height of bar shows (the logarithm of) the number of mutations for each sample,
and the ratio of colored division shows the ratio of estimated membership parameters for each signature and sample.
The low grade glioma specific signature detected by the proposed method is the signature 2.
We can see that the mutations corresponding to signature 2 is mostly from the sample with an extremely high mutation rate.
}

\end{figure}

\clearpage

\begin{figure}
\centering

\subfigure[The result for the original data ($K = 2$)]{%
  \includegraphics[height=2.5cm,width=15cm,clip]{Glioma-Low-Grade_original_signature_K2.eps}
  \label{LGG_roriginal_sigK2_sig}}
  
\subfigure[The result for the data without the hyper-mutated sample ($K = 2$)]{%
  \includegraphics[height=2.5cm,width=15cm,clip]{Glioma-Low-Grade_removed_signature_K2.eps}
  \label{LGG_removed_sigK2_sig}}
  
  
\subfigure[The result for the data without the hyper-mutated sample ($K = 3$)]{%
  \includegraphics[height=2.5cm,width=15cm,clip]{Glioma-Low-Grade_removed_signature_K3.eps}
  \label{LGG_removed_sigK3_sig}}
  

\subfigure[The result for the data without the hyper-mutated sample ($K = 4$)]{%
  \includegraphics[height=2.5cm,width=15cm,clip]{Glioma-Low-Grade_removed_signature_K4.eps}
  \label{LGG_removed_sigK4_sig}}
  
\subfigure[The result for the data without the hyper-mutated sample ($K = 5$)]{%
  \includegraphics[height=2.5cm,width=15cm,clip]{Glioma-Low-Grade_removed_signature_K5.eps}
  \label{LGG_removed_sigK5_sig}}
  
  \caption{(a) The signatures obtained for the original data. The first (from the left) signature seems to be deamination of 5-methyl-cytosine. 
  The second signature is the low grade glioma signature. 
  (b, c, d, e) The signature obtained for the data without the hyper-mutated case for several number of mutation signatures ($K$, including the background signature).
  Although the signature related to deamination of 5-methyl-cytosine remained, low grade glioma specific signature could not be observed.}

\end{figure}

\clearpage


\begin{figure}
\centering

\subfigure[The signature 2 detected in kidney clear cell carcinomas]{%
  \includegraphics[height=3cm,width=4.8cm,clip]{Kidney-Clear-Cell_oxidation_signature.eps}
  \label{LGG_membership}}
\quad
\subfigure[The signature 1 detected in lung adenocarcinomas]{%
  \includegraphics[height=3cm,width=4.8cm,clip]{Lung-Adeno_oxidation_signature.eps}
  \label{LGG_membership_log}}
\quad
\subfigure[The signature 1 detected in melanomas]{%
  \includegraphics[height=3cm,width=4.8cm,clip]{Melanoma_oxidation_signature.eps}
  \label{LGG_membership_log}}  
  
\subfigure[Estimated membership parameter for kidney clear cell carcinomas]{%
  \includegraphics[height=4cm,width=12cm,clip]{Kidney-Clear-Cell_oxidation_membership.eps}
  \label{LGG_nature2013_sig}}
  
\subfigure[Estimated membership parameter for lung adenocarcinomas]{%
  \includegraphics[height=4cm,width=12cm,clip]{Lung-Adeno_oxidation_membership.eps}
  \label{LGG_membership}}
\quad
\subfigure[Estimated membership parameter for melanomas]{%
  \includegraphics[height=4cm,width=12cm,clip]{Melanoma_oxidation_membership.eps}
  \label{LGG_membership_log}}

  \caption{(a, b, c) The putative oxidative artifact signatures estimated for each cancer.
  We can observe consistent abundance of the base C at the $-2 $ position.
  (d, e, f) For each cancer type, 
We have selected top 100 cancer samples according to the number of mutation.
The height of bar shows the number of mutations for each sample,
and the ratio of colored division shows the ratio of estimated membership parameters for each signature and sample.
We can see that the signature corresponding to putative oxidative artifacts concentrates on a small number of samples.}

\end{figure}



\end{document}

